test/API/H5_api_dataset_test.h

```diff
@@ -23,6 +23,13 @@ int H5_api_dataset_test(void);
  *                                              *
  ************************************************/
 
+/* Check if the relative difference is smaller than a
+ *  predefined value M.  See if two values are relatively equal.
+ *  It's the developer's responsibility not to pass in the value 0, which
+ *  may cause the equation to fail.
+ */
+#define VL_DBL_REL_EQUAL(X, Y, M) (fabs(((Y) - (X)) / (X)) < (M))
```

Comment:
Now that these tests will be back in the library, you should be able to use the equivalent `H5_DBL_REL_EQUAL` macro from H5private.h rather than creating a new one here.